### PR TITLE
Confirmation box comp

### LINKED ISSUE #669: Confirmation box component
Make reusable component.

1. use when sending single email

Used to both inform and to confirm if the user/admin wants to do an action:
* both Yes/No and Ok alternatives (for information)

### PR DESCRIPTION
**Pull request by havardMoe**
Confirmation and Information boxes working
image
image

will close #669

### CODE REVIEW

**Comment by havardMoe** [comment]
Could create a new issue for styling the components.. i.e. the buttons on the "confirmation" component-

File: ClientApp/src/components/shared/SendEmailContent.tsx [hunk 1]
```diff
@@ -30,6 +31,9 @@ const SendEmailContent: React.FC<ISendSingleEmail> = ({ open, handleClose, email
   const { getAccessTokenSilently } = useAuth0();
   const [userAccessToken, setUserAccessToken] = useState<string>("");
   const apiservice = new ApiService(userAccessToken);
+  // States for confirmation box
```
Comment (MagnusVG) [inline ClientApp/src/components/shared/SendEmailContent.tsx:34]: Remove comments

File: ClientApp/src/components/shared/SendEmailContent.tsx [hunk 2]
```diff
@@ -64,60 +69,75 @@ const SendEmailContent: React.FC<ISendSingleEmail> = ({ open, handleClose, email
       .then((response) => {
         if (response.status === 200) {
           handleClose();
+          // todo: Open dialog box here, all good
```
Comment (MagnusVG) [inline ClientApp/src/components/shared/SendEmailContent.tsx:72]: Remove comments

File: ClientApp/src/components/shared/SendEmailContent.tsx [hunk 3]
```diff
@@ -64,60 +69,75 @@ const SendEmailContent: React.FC<ISendSingleEmail> = ({ open, handleClose, email
...
+          setConfText("Sendte mail til: " + fullName + " <" + email + ">");
+          setOpenConfBox(true);
           setError(false);
           setErrorText("");
         }
       })
       .catch((errorStack) => {
         console.error(errorStack);
+        // todo: open dialog box here, not good
```
Comment (MagnusVG) [inline ClientApp/src/components/shared/SendEmailContent.tsx:81]: Remove comments

**Review by MagnusVG — APPROVED** [review]

**Review by MagnusVG — APPROVED** [review]

**Pull request merged**